src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValues.ts

```diff
@@ -57,7 +57,7 @@ export class CategoryFacetValues {
     return !!this.selectedPath.length;
   }
 
-  public reset() {
```

Comment:
Why is the rename needed here? I find both name good, but `reset` avoids a breaking change.

Reply:
It's a new file so it's not a breaking change (this is not the CategoryFacet file)